src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/failed.polling.aborts.pending.on-update.requests/accept/client.rpt

```diff
@@ -30,30 +30,49 @@ write nukleus:begin.ext ${http:header(":scheme", "http")}
                         ${http:header(":method", "GET")}
                         ${http:header(":path", "/data")}
                         ${http:header(":authority", "localhost:8080")}
-                        ${http:header("x-retry-after", "2")}
-                        ${http:header("if-modified-since", date)}
-                        ${http:header("if-none-match", etag)}
 connected
 
 write close
 
 read nukleus:begin.ext ${http:header(":status", "200")}
                        ${http:header("content-length", "11")}
-                       ${http:header("last-modified", dateIn5)}
-                       ${http:header("etag", etag2)}
-                       ${http:header("cache-control", "stale-while-revalidate=2147483648")}
+                       ${http:header("date", response1Date)}
+                       ${http:header("surrogate-control", "max-age=1+100")}
+                       ${http:header("etag", etag)}
+                       ${http:header("cache-control", "stale-while-revalidate=100")}
 
 read nukleus:data.ext ${http:header(":scheme", "http")}
                       ${http:header(":method", "GET")}
                       ${http:header(":path", "/data")}
                       ${http:header(":authority", "localhost:8080")}
-                      ${http:header("x-retry-after", "2")}
-                      ${http:header("if-modified-since", dateIn5)}
-                      ${http:header("if-none-match", etag2)}
-                      ${http:header("x-poll-injected", "x-http-cache-sync, no-cache")}
                       ${http:header("cache-control", "no-cache")}
-                      ${http:header("x-http-cache-sync", "always")}
+                      ${http:header("prefer", "x-on-update, wait=1")}
+                      ${http:header("if-none-match", etag)}
```

Comment:
`if-none-match` makes sense, but why would you also need the `x-on-update` preference?